docs/Home.md

```diff
@@ -0,0 +1,101 @@
+# Google Kubernetes Engine Plugin Wiki
+
+The [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) Plugin allows you to publish deployments built within Jenkins to your Kubernetes clusters running within GKE.
+
+_Plugin Information_
+
+View Google Kubernetes Engine [on the plugin site](https://plugins.jenkins.io/google-kubernetes-engine) for more information.
+
+## Usage
+
+### IAM Credentials
+
+1. Create a service account using the Google Cloud SDK.
+
+        gcloud iam service-accounts create jenkins-gke
+
+1. Add the KubernetesEngineAdmin, serviceAccountUser, and computeNetworkViewer roles to the service account:
+
+        export PROJECT=$(gcloud info --format='value(config.project)')
+        export SA_EMAIL=$(gcloud iam service-accounts list --filter="name:jenkins-gke" --format='value(email)')
+        gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/iam.serviceAccountUser $PROJECT
+        gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/container.clusterAdmin $PROJECT
+        gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/container.admin $PROJECT
+        gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/compute.networkViewer $PROJECT
+
+1. Download a JSON Service Account key for your newly created service account. Take note of where the file was created, you will upload it to Jenkins in a subsequent step.
+
+        gcloud iam service-accounts create jenkins-gke
```

Comment:
The command for this is: gcloud iam service-accounts keys create ~/jenkins-gke-key.json
  --iam-account jenkins-gke@${PROJECT}.iam.gserviceaccount.com

Reply:
Good catch thanks